docs/mobile-apps/features/network-capture.md

```diff
@@ -7,92 +7,182 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+<img src={useBaseUrl('img/mobile-apps/network-capture-1.png')} alt="Select Developer Options" width="700"/>
 
-    * Start the capture.
+4. Start the network capture.
    
-<img src={useBaseUrl('img/mobile-apps/network-capture-2.png')} alt="Start capture" width="650"/>
+<img src={useBaseUrl('img/mobile-apps/network-capture-2.png')} alt="Start capture" width="700"/>
 
-    * Inspect network logs as they arrive
+5. Inspect network logs as they arrive.
  
-<img src={useBaseUrl('img/mobile-apps/network-capture-3.png')} alt="Inspect network logs" width="650"/>
-
-    * A copy of what you see in the inspector can be downloaded in HAR format with the following button.
+<img src={useBaseUrl('img/mobile-apps/network-capture-3.png')} alt="Inspect network logs" width="700"/>
+
+### Sauce Labs Network Viewer
+The [Sauce Labs Network Viewer](https://github.com/saucelabs/network-viewer) is a visualization tool for HAR files that presents requests
+in a table view with a range of advanced filtering options.
+
+#### Pause / Resume
+Pause and resume the display of new network traffic without affecting the **network.har** test asset.
+The final **network.har** file contains all network traffic from the moment the network capture was initiated and will be available after the
+session is closed.
+<img src={useBaseUrl('img/mobile-apps/network-capture-pause-resume.png')} alt="Inspect network logs" width="700"/>
+
+#### Reset
+Clear the current state of network logs from the inspector. This does not affect the final **network.har** test asset.
+<img src={useBaseUrl('img/mobile-apps/network-capture-reset.png')} alt="Inspect network logs" width="700"/>
+
+#### Export HAR file
+Download the current state of network logs from the inspector in HAR format. This is helpful for capturing a specific part of the test flow
+with a network issue, making it easier to share with developers for debugging.
+<img src={useBaseUrl('img/mobile-apps/network-capture-export.png')} alt="Inspect network logs" width="700"/>
+
+#### Filter by status
+Use the status filter dropdown list to filter network logs based on their status code.
+<img src={useBaseUrl('img/mobile-apps/network-capture-status-filter.png')} alt="Inspect network logs" width="700"/>
+
+#### Search by URL
+Search for specific network logs using the request URL.
+<img src={useBaseUrl('img/mobile-apps/network-capture-search.png')} alt="Inspect network logs" width="700"/>
+
+#### Filter by content type
+Filter requests by their content type by selecting the desired category from the options available.
+<img src={useBaseUrl('img/mobile-apps/network-capture-type-filter.png')} alt="Inspect network logs" width="700"/>
+
+#### Requests table
+The main table displays all captured requests with the following columns:
+* Path: The name of the request. Hover over it to view the full resource, including the domain name.
+* Domain: The domain of the request.
+* Status: The HTTP status code.
```

Comment:
so lets call everywhere HTTP/S so nobody can call it out that why we support only HTTP